scripts/docker-entrypoint.sh

```diff
@@ -143,11 +141,21 @@ fi
 #
 if [ -e "$prconfig" ]; then
   echo "Loading prconfig from ${prconfig}...";
-  cp "$prconfig" ${CATALINA_HOME}/webapps/prweb/WEB-INF/classes/
+  cp "$prconfig" /opt/pega/prweb/WEB-INF/classes/
```

Comment:
I am concerned that this change is not backwards-compatible, can we do some kind of check to see if /opt/pega/prweb exists, and if not default to the previous logic (both here and in other similar locations)?

Reply:
@Emilio-Pega You are correct.  This change is not backwards compatible.  However, I'm thinking that it may make more sense to make use of the docker image version to assure backwards compatibility instead of muddying the code to support multiple implementations.  The version of the pega-web-ready image was updated in this PR for that purpose.  I'm thinking that we can (and probably should regardless) update the various official pega images (e.g. docker-pega, docker-pega-packedengine, others?) to use version 2.1 as the base.  This would reduce the possibility of breaking upstream consumers of our base images (e.g. SDE as you pointed out in my other PR). To me this seem like a cleaner approach.   Thoughts?

Reply:
I think, the issue is, that the prpc-paltform CI builds use whatever the 'latest' is from the https://git.pega.io/projects/PRPC/repos/docker-pega/browse during each build (not just on latest master but in older release branches as well) which is then consumed by the SDE images

So introducing non-compatible changes means that, internally, a lot of stuff is going to end up breaking (local dev, sdes, maybe CI builds) until the appropriate changes are made on each of the prpc-platform branches.

So if you go forward with breaking changes you need to be super careful internally.  For example:
* need to tell us to halt SDE update
* need to tell devs that local dv might be broken while the process is ongoing
* etc...

Reply:
We have also been told by @dcasavant (and others) that they generally want the SDEs on the 'latest' images

Reply:
Maybe this change can be made in stages?

Stage 1 - feature with backwards compatibility fallback: We can then fix all the internal downstream consumers
Stage 2 - remove backwards compatibility

Reply:
I have some thoughts about some changes that I can make here to better support the old and new locations, but this (and changes that allow both locations) would still require changes upstream in dependent images.

Reply:
Although I think it would be better for upstream images to make use of the image version to account for changes such as this, I have updated the logic here to move the pega deployment back to tomcat/webapps if the application context is prweb.  This should accommodate upstream images that either deploy to tomcat/webapps directly or assume the deployment is in tomcat/webapps.